pipeline/vars/lib.groovy

```diff
@@ -366,7 +366,7 @@ def sendEmail(
         status = "UNSTABLE"
     }
 
-    def subject = "${subjectPrefix} ${tierLevel.capitalize()} test report status of ${artifactDetails.version} - ${artifactDetails.ceph_version} is ${status}"
+    def subject = "[Nightly Pipeline] ${tierLevel.capitalize()} test report status of ${artifactDetails.version} - ${artifactDetails.ceph_version} is ${status}"
```

Comment:
This change would be break `[Cron Pipeline]` subject line.

Reply:
Done.